content/2018-09-25-newsletter-12.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "The Embedded Working Group Newsletter - 12"
+date = 2018-09-25
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the twelfth newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [internals.rust-lang.org], [on twitter], or [on reddit]!
```

Comment:
Perhaps we should post this to u.r-l.o under the embedded category?

Reply:
+1

Reply:
Will change